gui/wxpython/mapwin/base.py

```diff
@@ -95,6 +95,23 @@ def alignExtent(self, value):
             self.alignExtentChanged.emit(value=value)
 
 
+class StatusBarProperties(object):
```

Comment:
Seems this could be part of MapWindowProperties, no?

Reply:
Well.. I had the feeling that it should be separated because status bar properties do not influence a map window itself in any way. But it is probably early to decide, we can divide it later on if needed.